### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krossovok83**

### CODE REVIEW

File: lib/class/library.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  def initialize
+    generate
+  end
+
+  def self.load
+    File.exist?(LIBRARY) ? YAML.load_file(LIBRARY) : new
+  end
+
+  def generate
+    @books = (1..50).map { Book.generate }
```
Comment (papandreusa) [inline lib/class/library.rb:18]: Pls make contants

File: lib/module/stat.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Calculate statistics for library
```
Comment (papandreusa) [inline lib/module/stat.rb:3]: Remove comments, pls

File: lib/module/stat.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  def best_books(number = 1)
+    best_book = orders.map { |order| order.order_book.title }
+    best_book.group_by { |title| title }.transform_values(&:count).sort_by { |_key, value| -value }.first(number)
+  end
+
+  def some_quantity(number = 3)
+    count = 0
+    best_books(number).map!(&:first).each do |book_title|
+      ordered_best_books = orders.select { |order| order.order_book.title == book_title }
+      count += ordered_best_books.map { |order| order.order_reader.reader_name }.uniq.length
```
Comment (papandreusa) [inline lib/module/stat.rb:19]: Use reduce method, pls

**Review by papandreusa — CHANGES_REQUESTED** [review]

File: lib/class/library.rb (hunk 1 above)
Comment (krossovok83) [reply lib/class/library.rb]: fix

File: lib/module/stat.rb (hunk 2 above)
Comment (krossovok83) [reply lib/module/stat.rb:3]: fix

File: lib/module/stat.rb (hunk 3 above)
Comment (krossovok83) [reply lib/module/stat.rb]: done

**Pull request closed without merging**